rastervision_pytorch_learner/rastervision/pytorch_learner/utils/torch_hub.py

```diff
@@ -136,22 +118,21 @@ def torch_hub_load_uri(uri: str, hubconf_dir: str, entrypoint: str,
     is_zip = uri_path.suffix.lower() == '.zip'
     if is_zip:
         # unzip
-        zip_path = download_if_needed(uri, tmp_dir)
-        unzip_dir = join(tmp_dir, uri_path.stem)
-        _remove_dir(unzip_dir)
-        unzip(zip_path, target_dir=unzip_dir)
-        unzipped_contents = list(glob(f'{unzip_dir}/*', recursive=False))
-
-        _remove_dir(hubconf_dir)
-
-        # if the top level only contains a directory
-        if (len(unzipped_contents) == 1) and isdir(unzipped_contents[0]):
-            sub_dir = unzipped_contents[0]
-            shutil.move(sub_dir, hubconf_dir)
-        else:
-            shutil.move(unzip_dir, hubconf_dir)
-
-        _remove_dir(unzip_dir)
+        zip_path = download_if_needed(uri)
+        with TemporaryDirectory() as tmp_dir:
```

Comment:
I think this won't respect the temporary directory that is set in `RVConfig`. Should you use `RVConfig.get_temp_dir()`?

Reply:
This is just for initially extracting the zip. The extracted contents are eventually moved to `hubconf_dir` (line 133 and 135) and this `tmp_dir` is destroyed immediately after. So it doesn't really where the `tmp_dir` is created.

Reply:
It could matter if the zip file is huge and extracting it in the system provided temp dir will cause it to run out of disk space. The Docker container file system only has 22GB in it, so that's why we mount the `/opt/data` directory to the host file system which has more space, and then put the temporary directories in `/opt/data`.

Reply:
That makes sense but there are also other places where we are not using a tmp dir rooted in `RVConfig`'s tmp dir. I guess we should change those too.